VideoLocker/src/main/java/org/edx/mobile/view/EditUserProfileFragment.java

```diff
@@ -161,18 +170,9 @@ public boolean onMenuItemClick(MenuItem item) {
                                 break;
                             }
                             case R.id.remove_photo: {
-                                executePhotoTask(new DeleteAccountImageTask(getActivity(), username) {
-                                    @Override
-                                    protected void onSuccess(Void aVoid) throws Exception {
-                                        hideLoading();
-                                    }
-
-                                    private void hideLoading() {
-                                        if (null != viewHolder) {
-                                            viewHolder.profileImageProgress.setVisibility(View.GONE);
-                                        }
-                                    }
-                                });
+                                final Task task = new DeleteAccountImageTask(getActivity(), username);
```

Comment:
`DeleteAccountImageTask ` can be converted into a `Call`, consequently, making the function `executePhotoTask ` accept a `Call`type object.